frontend/src/components/pages/CampCreation/ScheduleSessions/SidePanel/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import AddSessionsForm from "./AddSessionsForm";
+import CurrentSessionsView from "./CurrentSessionsView";
+import { CreateCampSession } from "../../../../../types/CampsTypes";
+
+type SessionSidePanelProps = {
+  scheduledSessions: CreateCampSession[];
+  setScheduledSessions: (sessions: Array<CreateCampSession>) => void;
+};
+
+const SessionSidePanel = ({
+  scheduledSessions,
+  setScheduledSessions,
+}: SessionSidePanelProps): React.ReactElement => {
+  const [showAddSessionView, setShowAddSessionView] = React.useState(false);
+
+  return (
+    <Box minWidth="35vw" h="100%" background="background.grey.200" zIndex="10">
+      {scheduledSessions.length === 0 || showAddSessionView ? (
+        <AddSessionsForm
```

Comment:
not part of this ticket, but do you think we can make the cards for the sessions a little smaller? On my 13 inch mac, it looks like this:
<img width="1376" alt="image" src="https://user-images.githubusercontent.com/57159970/202932649-726bc492-f222-45ca-b3a5-68e030b2784b.png">

I think 13 inch computers should be a common screen size?